src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -45,6 +45,7 @@ const buttonStyles: IComponentPartStylesInput = {
         (iconPosition
           ? {
               display: 'inline-flex',
+              alignItems: 'center',
```

Comment:
as discussed, there is a corresponding PR #135 where bunch of related `Button` problems are addressed. 

As a next work item related to that - the PR with conformance tests that will ensure that shorthand prop names correspond to component's anatomy parts, so that it would be intuitive for client to decide which name should be used for styling specific component's part:

```jsx
<Button icon={...} styles={ icon: () => ... } />
```